mistletoe/markdown_renderer.py

```diff
@@ -0,0 +1,542 @@
+"""
+Markdown renderer for mistletoe.
+"""
+
+import re
+from itertools import chain
+from typing import Iterable, Sequence
+
+from mistletoe import block_token, span_token, token
+from mistletoe.base_renderer import BaseRenderer
+
+
+class BlankLine(block_token.BlockToken):
+    """
+    Blank line token. Represents a single blank line.
+    This is a leaf block token without children.
+    """
+
+    pattern = re.compile(r"\s*\n$")
```

Comment:
Just a detail, but can we please have `'` instead of `"`, generally? `'` seems to be the prevailing variant, and I would like to make the code base consistent in this regard in the near future.

Reply:
I have absolutely no preferences about code formatting issues like this, I just like a consistent code base. That's why I'm fond of the [black](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings) formatter, and the preference there happens to be double quotes.

So, I can change to single quotes, but before I do that: are you sure you want to standardize on single quotes and not double quotes? From what I can see, black is the only popular formatter that standardizes the quotes, and it chose the other alternative.

Reply:
Well, you forced me to do some extra research again, but in the good way of course. :)

TBH, I'm not still quite sure about this decision. The dilemma is nicely described for example in <https://medium.com/pythoniq/so-single-or-double-quotes-in-python-47c2e7425f32>. Anyway, as I primarily "live" in he Java world, I think I have no problem with using **double quotes** everywhere. Also, as you mentioned, it is the _default_ choice in the popular Black formatter (i.e. it should be a switchable option when needed).

Reply:
With _black_ you only have the option to disable string formatting; the preferred string separator is double quotes and that's it. But ... it turns out that enough people have seen the need for the option, so there is now also a black derivative called _blue_ where the default is single quotes. So I'll reformat with blue, and that should do the trick.

Reply:
Maybe I jumped to conclusions too fast there. But it's easy to back out that commit again, should you decide that double quotes is the way to go.

Reply:
Yeah, I think I'm OK with _black_'s double quotes for now. :)

Reply:
ok! I've backed out the reformatting commit.

Reply:
OK, thank you.